MyForage/MyForage/ViewControllers/AddForageVC.swift

```diff
@@ -28,6 +28,7 @@ class AddForageVC: UIViewController {
     // MARK: - Properties
     
     weak var coordinator: MainCoordinator?
+    let apiController: ApiController = ApiController()
```

Comment:
The apiController should be removed from this file, since it is now instantiated in the coordinator.

Reply:
```suggestion
    
```